src/Sentry/Internal/FastSerialization/GrowableArray.cs

```diff
@@ -347,11 +347,6 @@ private void Realloc(int minSize)
     // Implementation of IEnumerable<T>.
     public IEnumerator<T> GetEnumerator() => new GrowableArrayEnumerator(this);
 
-    internal void Add(object value)
```

Comment:
This GrowableArray is part of the `perfview` module. Can we use that implementation isntead?

Reply:
We can't because it's only depended up in `Sentry.Profiling` project and this is part of `Sentry` (so it can be used directly in protocol classes)